src/legacy/index.ts

```diff
@@ -251,8 +251,8 @@ function constructPackageFormatVersion(pkgType: string): string {
 }
 
 function constructTargetOS(depGraph: types.DepGraph): { name: string; version: string; } | void {
-  if (['apk', 'apt', 'deb', 'rpm'].indexOf(depGraph.pkgManager.name) === -1) {
-    // .targetOS is undefined unless its a linux pkgManager
+  if (['apk', 'apt', 'deb', 'rpm', 'linux'].indexOf(depGraph.pkgManager.name) === -1) {
+    // when importing container images built from scratch, pgkManager will be linux
```

Comment:
wouldn't change this comment 🤔  
or at least not remove the previous one

Reply:
Maybe I misunderstood, but  isn't it not true anymore?

Reply:
we still don't create `targetOS` for npm/ruby/pip etc. ..

Reply:
You're right.. changed 🙈